src/xcode/ENA/ENA/Source/Protocols/CollectionViewCellConfigurator.swift

```diff
@@ -22,10 +22,10 @@ protocol CollectionViewCellConfiguratorAny: AnyObject {
 
 	func configureAny(cell: UICollectionViewCell)
 
-	var identifier: UUID { get }
+	var hashValue: AnyHashable { get }
```

Comment:
I believe this is needed because of type erasure, correct?

Reply:
you're right